api/response/schema_query.py

```diff
@@ -41,14 +48,18 @@ class Query(graphene.ObjectType):
         PaginatedResponses,
         page=graphene.Int(),
         per_page=graphene.Int(),
-        upper_limit=graphene.Int(),
-        lower_limit=graphene.Int(),
+        upper_limit_count=graphene.Int(),
+        lower_limit_count=graphene.Int(),
+        end_date=graphene.String(),
+        start_date=graphene.String(),
         room=graphene.String(),
         description="Returns a list of room responses. Accepts the arguments\
             \n- page: Page number of responses\
             \n- per_page: Number of room responses per page\
-            \n- upper_limit: Highest number of room responses\
-            \n- lower_limit: Highest number of room responses\
+            \n- upper_limit_count: Highest number of room responses\
+            \n- lower_limit_count: Highest number of room responses\
```

Comment:
Should this be the highest or lowest number of room responses?

Reply:
Thanks for pointing this out @KMaina, I've made the necessary changes